sample_apps/simple_app/example_app.mak

```diff
@@ -28,12 +28,13 @@ S= $(CERTIFIER_ROOT)/src
 O= $(OBJ_DIR)
 US=.
 I= $(CERTIFIER_ROOT)/include
-INCLUDE= -I$(I) -I/usr/local/opt/openssl@1.1/include/ -I$(S)/sev-snp/
+INCLUDE= -I$(I) -I/usr/local/opt/openssl@1.1/include/ -I$(S)/sev-snp/ -I.
```

Comment:
Minor. I would think that the `-I.`; i.e. include current-dir would conventionally be 1st in the list, rather than the last one. No?